packages/configs/default/index.json

```diff
@@ -40,6 +40,7 @@
     "style:*.vue": ["@parcel/transformer-vue"],
     "custom:*.vue": ["@parcel/transformer-vue"],
     "*.{png,jpg,jpeg,webp}": ["@parcel/transformer-image"],
+    "*.{atom,rss,svg,xml}": ["@parcel/transformer-xml"],
```

Comment:
Seems like we should probably have different transformers for each of these. Atom, RSS, and SVG will all have different tags and attributes that we'll need to handle to create dependencies for sub resources. Maybe keep this transformer specific to `.xml` files?